evm/generators.go

```diff
@@ -793,6 +795,8 @@ func AddCLI(sourceCode, structName string, noformat, includemain bool) (string,
 					&ast.ImportSpec{Path: &ast.BasicLit{Value: `"github.com/ethereum/go-ethereum/accounts/keystore"`}},
 					&ast.ImportSpec{Path: &ast.BasicLit{Value: `"github.com/ethereum/go-ethereum/ethclient"`}},
 					&ast.ImportSpec{Path: &ast.BasicLit{Value: `"golang.org/x/term"`}},
+					&ast.ImportSpec{Path: &ast.BasicLit{Value: `"github.com/moonstream-to/seer/bindings/GnosisSafe"`}},
```

Comment:
I think it's ok to have this for now. Eventually, though, I would like to move all Safe-related library functionality to the [`G7DAO/safes`](https://github.com/G7DAO/safes) repo and make that the dependency.

Reply:
I will create an issue for that in both repositories

Reply:
Issues created: 

- https://github.com/moonstream-to/seer/issues/74
- https://github.com/G7DAO/safes/issues/1